gateway/response_test.go

```diff
@@ -97,11 +97,11 @@ func TestForwardResponseMessage(t *testing.T) {
 		t.Fatalf("failed to unmarshal JSON response: %s", err)
 	}
 
-	if v.Success["code"] != CodeName(Created) {
+	if v.Success["status"] != CodeName(Created) {
 		t.Errorf("invalid status code: %s - expected: %s", v.Success["code"], CodeName(Created))
 	}
 
-	if v.Success["status"].(float64) != http.StatusCreated {
+	if v.Success["code"].(float64) != http.StatusCreated {
```

Comment:
Double check this. Look at the line below as it is giving error for v.Success["status"] and is checking v.Success["code"]